docs/resources/yaml.md.erb

```diff
@@ -11,36 +11,36 @@ Use the `yaml` InSpec audit resource to test configuration data in a Yaml file.
 
 ## Syntax
 
-A `yaml` resource block declares the configuration data to be tested. Assume the following Yaml file:
+A `yaml` resource block declares the configuration data to be tested. Assume the following Yaml file called 'filename.yml':
 
```

Comment:
Keep "Assume the following YAML file:" -- the  filename is part of what is assumed.

Reply:
The file name is not mentioned anywhere and then one is used below in the example. I thought it would be useful to see where the filename came from but fine with changing it back.